ownCloud/Resources/en.lproj/Localizable.strings

```diff
@@ -406,3 +406,7 @@
 "Delete Offline Copies" = "Delete Offline Copies";
 "Manage" = "Manage";
 "Storage" = "Storage";
+
+/* Import File */
+"Save File" = "Save File";
+"Select an account where to import the file and choose a destination directory." = "Select an account where to import the file and choose a destination directory.";
```

Comment:
Just an alternative suggestion: `Choose an account and directory to import the file into.`

Reply:
Used one of your description